packages/datadog_flutter_plugin/doc/rum/otel_support.md

```diff
@@ -0,0 +1,39 @@
+# RUM Flutter Open Telemetry Support
+
+The [Datadog Tracking Http Client][1] package and [gRPC Interceptor][2] package both support distributed traces through both automatic header generation and header ingestion.
```

Comment:
Header ingestion?

Reply:
Good catch, I wrote this before I removed the ability for the client to parse and forward OTel headers.